accounting/receivables/customer_invoices/payment_terms.rst

```diff
@@ -1,96 +1,109 @@
-+----------------------+------------+---------+----------+
+In the **Terms** section, you can add a set of rules, that we call *terms*, to define what needs to
+be paid, and by which due date.
 
-If you do an invoice the 1st of January with a payment term of 10%
-within 3 days and the balance within 30 days, you get the following
-journal entry:
+To add a term, click on *Add a line*, and define its *Type*, *Value*, and *Due Date Computation*.
 
-+----------------------+------------+---------+----------+
-| Account              | Due date   | Debit   | Credit   |
-+======================+============+=========+==========+
-| Account Receivable   | Jan 03     | 10      |          |
-+----------------------+------------+---------+----------+
-| Account Receivable   | Jan 30     | 90      |          |
-+----------------------+------------+---------+----------+
-| Income               |            |         | 100      |
-+----------------------+------------+---------+----------+
+.. important::
+   - Terms are computed in the order they are set up.
+   - The **balance** should always be used for the last line.
 
-In the customer statement, you will see two lines with different due
-dates.
+In the following example, 30% of the invoice is due on the day of issuance of the invoice, and the
+balance is due at the end of the following month.
 
-Payment terms for vendor bills
-------------------------------
+.. image:: media/payment_terms_configuration.png
+   :align: center
+   :alt: Example of payment terms. The last line is the balance due the 31st of the following month.
 
-The easiest way to manage payment terms for vendor bills is to record a
-due date on the bill. You don't need to assign a payment term, just the
-due date is enough.
+Using Payment Terms
+===================
 
-But if you need to manage vendor terms with several installments, you
-can still use payment terms, exactly like in customer invoices. If you
-set a payment term on the vendor bill, you don't need to set a due date.
-The exact due date for all installments will be automatically created.
+Payment Terms can be defined with the **Payment Terms** field on:
+
+- | **Contacts**
+  | To set specific payment terms automatically on new sales orders, invoices, and bills of a
+    contact. This can be modified in the contact’s *Form View*, under the *Sales & Purchase* tab.
+- | **Quotations**
+  | To set specific payment terms automatically on all invoices generated from a quotation.
+- | **Customer Invoices**
+  | To set specific payment terms on an invoice.
+- | **Vendor Bills**
+  | To set specific payment terms on a bill. This is mostly useful when you need to manage vendor
+    terms with several installments. Otherwise, setting the *Due Date* is enough.
+
+Journal Entries
+===============
+Invoices with specific Payment Terms generate different *Journal Entries*, with one *Journal Item*
+for each different *Due Date* computed.
```

Comment:
```suggestion
  for each different *Due Date* computed.
```

Reply:
and the same for all sections when you have more than 1 paragraph, as you want to break the line but not the paragraph.

Reply:
The following paragraph isn't correctly displayed with a separation, after block lines, in our documentation.
Also, I usually prefer to have one idea per paragraph, in English. And weirdly, I regroup ideas with line breaks in French... I don't know why.

So for now, I'll keep it this way, but I might change it later.

Cheers and thanks.